tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2803,6 +2803,23 @@ TEST_P(OpConverter_FP32_Test, ConvertBatchMatMul) {
                    return p;
                  });
 
+  // 4D tensor where the second "batch dim" is not 1
+  params.push_back(MatMulTestParams{
+      {1, 2, 4, 5},
+      {0.,  1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.,  9.,  10., 11., 12., 13.,
+       14., 15., 16., 17., 18., 19., 20., 21., 22., 23., 24., 25., 26., 27.,
+       28., 29., 30., 31., 32., 33., 34., 35., 36., 37., 38., 39},
+      false,  // A
+      {1, 2, 3, 5},
+      {1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.,  9.,  10.,
+       11., 12., 13., 14., 15., 16., 17., 18., 19., 20.,
+       21., 22., 23., 24., 25., 26., 27., 28., 29., 30.},
+      true,  // B
+      {1, 2, 4, 3},
+      {40.,   90.,   140.,  115.,  290.,  465.,  190.,  490.,
+       790.,  265.,  690.,  1115., 1990., 2540., 3090., 2440.,
+       3115., 3790., 2890., 3690., 4490., 3340., 4265., 5190.}});  // result
```

Comment:
I am getting this error messages:
third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc:2801:8: error: type 'double' cannot be narrowed to 'int' in initializer list [-Wc++11-narrowing]
      {0.,  1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.,  9.,  10., 11., 12., 13.,
       ^~
third_party/tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc:2801:8: note: insert an explicit cast to silence this issue
      {0.,  1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.,  9.,  10., 11., 12., 13.,
       ^~
       static_cast<int>( )

Reply:
Sorry for that. I have resolved the problem.